core-bundle/src/Twig/Inheritance/DynamicIncludeTokenParser.php

```diff
@@ -54,9 +55,23 @@ public function getTag(): string
     /**
      * Return the adjusted logical name or the unchanged input if it does not
      * match the Contao Twig namespace.
+     *
+     * TODO change signature to public static function adjustTemplateName(string|TemplateWrapper $name, TemplateHierarchyInterface $hierarchy): string|TemplateWrapper in Contao 5.1
```

Comment:
5.1?

Reply:
This TODO meant for the upstream merge I think.